content/en/real_user_monitoring/browser/tracking_user_actions.md

```diff
@@ -20,11 +20,23 @@ further_reading:
 ---
 
 
-Real User Monitoring (RUM) SDKs detect user interactions performed during a user journey. Set the `trackInteractions` [initialization parameter][1] to `true` to enable this feature.
+Real User Monitoring (RUM) Browser SDK automatically detects user interactions performed during a user journey. 
 
-**Note**: The `trackInteractions` initialization parameter enables the collection of user clicks in your application. **Sensitive and private data** contained on your pages may be included to identify the elements interacted with.
+The automatic collection of user actions provides insights into user behavior, without having to instrument every single click in your application. It will help you achieve the following objectives:
```

Comment:
> without having to instrument every single click

We instrument every click, but we retain only the significant ones. Maybe something like:

> The automatic collection of user actions provides insights into user behavior, by retaining only significant clicks in your application

Reply:
added an extra note, **they** don't have to manually instrument their applications